source/documents/delete-multiple.txt

```diff
@@ -14,4 +14,134 @@ Delete Multiple Documents
+
+      From the :guilabel:`Documents` tab, input a query into the 
+      :guilabel:`Query bar`. The filter criteria of the query specified 
+      applies to the documents in the :guilabel:`Delete Documents` 
+      modal. If you need to delete all documents in a collection, 
+      leave the :guilabel:`Query bar` blank.
+
+   .. step:: Open the delete documents modal
+
+      On the :guilabel:`Documents` tab, click the :icon-lg:`Trash` 
+      :guilabel:`Delete` button to display the 
+      :guilabel:`Delete Documents` modal. The following 
+      table summarizes the :abbr:`UI (User Interface)` of the modal:
+
+      .. list-table::
+         :widths: 40 60
+         :header-rows: 1
+
+         * - UI Element
+           - Description
+
+         * - :guilabel:`Query`
+           - Any filter criteria specified on the 
+             :guilabel:`Query Bar` applies to the 
+             :guilabel:`Delete Documents` modal.
+             To update the :guilabel:`Query`, exit the 
+             :guilabel:`Delete Documents` modal and modify the 
+             query in the :guilabel:`Query Bar`.
+         * - :guilabel:`Export`
+           - Prompts the :guilabel:`Export Delete Query To Language`
+             modal. The :guilabel:`Export Delete Query To Language` 
+             modal allows you to convert the delete query
+             to a supported driver language.
+         * - :guilabel:`Preview`
+           - A preview of the documents that will be deletion.
+
+   .. step:: (Optional) Export the :guilabel:`Delete`
+
+      You can export the :guilabel:`Delete` query to a 
+      supported driver language using the :guilabel:`Export` button on 
+      the :guilabel:`Delete Documents` modal. 
+
+      a. On the :guilabel:`Delete Documents` modal, click 
+         :guilabel:`Export`. The 
+         :guilabel:`Export Delete Query To Language` modal displays with
+         the delete syntax populated under :guilabel:`My Delete Query`.
+
+      #. Select a programming language by clicking the drop-down under 
+         :guilabel:`Exported Delete Query`. You can convert your delete 
+         syntax to: C#, Go, Java, Node, PHP, Python, Ruby, and Rust. The 
+         converted delete syntax displays under the 
+         :guilabel:`Exported Delete Query` header.
+
+      #. (Optional) Include import statements 
+         Click the :guilabel:`Include Import Statements` checkbox to 
+         include the required import statements for the selected 
+         programming language.
+
+      #. Click the :icon-lg:`Copy` icon to copy the converted syntax.
+
+      #. Click the :guilabel:`Close` button.
+
+   .. step:: Delete your documents
+
+      a. On the :guilabel:`Delete Documents` modal, click 
+         :guilabel:`Delete Documents`.
+      #. Click the red :guilabel:`Delete Documents` button to confirm
+         the operation.
+
+      Compass applies the :guilabel:`Delete` to the documents
+      within the :guilabel:`Filter` expression.
+
+Example
```

Comment:
Not actually sure we need an example here, the procedure does a decent job of covering the workflow. Curious for feedback if this adds value.